packages/ag-charts-community/src/scale/logScale.ts

```diff
@@ -33,6 +33,18 @@ function pow(base: number, domain: number[], x: number) {
     return start >= 0 ? fn(base, x) : -fn(base, -x);
 }
 
+function transform(this: LogScale, x: number) {
+    const [min, max] = findMinMax(this.domain);
```

Comment:
Or - if you took the domain in here, we could avoid `this` references

Reply:
Ditto